specification/chaos/resource-manager/Microsoft.Chaos/preview/2022-10-01-preview/types/experiments.json

```diff
@@ -351,6 +355,25 @@
       ],
       "additionalProperties": false
     },
+    "filters": {
+      "description": "Model that represents the Experiment properties model.",
+      "type": "object",
+      "properties": {
+        "zones": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/zone"
+          },
+          "description": "List of Azure availability zones to filter targets by.",
+          "minItems": 0
+        }
+      },
+      "additionalProperties": false
+    },
+    "zone": {
+      "type": "string",
+      "description": "Azure availability zone to filter targets by."
```

Comment:
Follow existing pattern for descriptions: "String of the availability zone ID."

Reply:
Updated